src/sentry/incidents/logic.py

```diff
@@ -566,8 +566,10 @@ def create_alert_rule(
     """
     if monitor_type == AlertRuleMonitorTypeInt.ACTIVATED and not activation_condition:
         raise ValidationError("Activation condition required for activated alert rule")
-
-    resolution = get_alert_resolution(time_window, organization)
+    if detection_type == AlertRuleDetectionType.DYNAMIC:
+        resolution = time_window
```

Comment:
If the concern is that we're sending _too_ much data too frequently

then rather than sending _too much_ data less frequently, Maybe we should be sending less data more frequently?

Reply:
Yes, right now the smallest granularity we have is 15 minutes, and introducing a 5 minute granularity should be technically feasible and help address your concern